build-settings/rollup.config.js

```diff
@@ -58,10 +58,10 @@ const createConfig = ({name, format, platform, file}) => ({
                 {
                     // src path is relative to the package root unless started
                     // with ./
-                    src: "build-settings/index.flow.js.template",
+                    src: "build-settings/index.js.flow.template",
                     // dest path is relative to src path.
                     dest: makePackageBasedPath(name, "./dist"),
-                    rename: "index.flow.js",
+                    rename: "index.js.flow",
```

Comment:
This was a typo. As soon as I started trying to get flow working across packages, I realised my error.